src/HydraAuction/OnChain/Common.hs

```diff
@@ -60,6 +62,11 @@ checkVoucherExpiredOrLater terms info =
   traceIfFalse "Wrong interval for transaction" $
     contains (from (voucherExpiry terms)) (txInfoValidRange info)
 
+-- This function is not used on-chain
+secondsLeftInInterval :: Integer -> Interval POSIXTime -> Maybe Integer
```

Comment:
I think either this should get haddock or the first parameter's type should be changed to be a type that represents time, especially as this makes assumptions about the type of argument that we get passed. Ideally the type of the first argument should be changed as well.